care/facility/models/asset.py

```diff
@@ -177,3 +182,32 @@ class AssetTransaction(BaseModel):
     performed_by = models.ForeignKey(
         User, on_delete=models.PROTECT, null=False, blank=False
     )
+
+
+class AssetService(BaseModel):
+    asset = models.ForeignKey(Asset, on_delete=models.PROTECT, null=False, blank=False)
+
+    serviced_on = models.DateField(default=None, null=True, blank=False)
+    note = models.TextField(default="", null=True, blank=True)
+
+    edits = models.ManyToManyField("AssetServiceEdit", related_name="edits")
```

Comment:
Why is this many to many?

Reply:
![image](https://github.com/coronasafe/care/assets/3626859/c935e470-16a7-4941-8a2a-8096fb44e5f5)

Resolved.